sklearn/metrics/regression.py

```diff
@@ -518,16 +518,16 @@ def r2_score(y_true, y_pred, sample_weight=None,
     >>> r2_score(y_true, y_pred, multioutput='variance_weighted')
     ... # doctest: +ELLIPSIS
```

Comment:
Could you please also move this line to the end of one of the lines above so that we don't get the empty `... ` line in the output?